### ISSUE TITLE
Test new normalization primers for geoduck repro qPCR

### ISSUE THREAD

**Issue by shellytrigg** [RobertsLab/resources#988]
@kubu4 can you test the new normalization primer pairs when they arrive (just added them to the [primer database](https://docs.google.com/spreadsheets/d/14m2kkFhxcoKWWIGoAD_7VOVsAg9wilME2UcSLqfnqLI/edit?authkey=CP736rQK#gid=0) this afternoon)? 

There are 4 new pairs, two pairs per gene as we discussed in lab meeting. They should amplify RPL5 which is a 60S ribosomal protein (PGEN_.00g287540-vv0.74.a) and TIF3 subunit 6b (PGEN_.00g000750-vv0.74.a) which is a translation initial factor 3 subunit. 

I'm thinking test these first on NTC and the positive control pool cDNA, then decide which performs best and use that for repeating the pilot qPCRs with APLP. What do you think?

Here are the new primer sequences below for reference:

```
>RPL5_v2_FWD
GCGTATGCCAGAATCGAGGG
>RPL5_v2_REV
CCTGTGCAGTAAGCAGCAGC
>RPL5_v3_FWD
GACGGACAGCCAGGAGCC
>RPL5_v3_REV
CAGACCACCATCGGCAGC
>TIF3s6b_v2_FWD
GGTGACACGCTTGTGTTTCAGG
>TIF3s6b_v2_REV
CACATTCCCGTAGCTTCTGC
>TIF3s6b_v3_FWD
GATGGCGCCCAGCAGAAG
>TIF3s6b_v3_REV
GGAGGTCATCCCTTGAAGGC
```

**Comment by kubu4** [comment]
All primer sets look good using pooled cDNA (e.g. good Cq, single peak in melt curve, etc).

All primer sets amplify gDNA; just something to be aware of.

Will run on existing, individual cDNA samples. However, just wondering if it's OK to just run on the "numbered" samples. The STAR/CHEW samples will not have sufficient cDNA to run all of these primer sets. Are the "numbered" samples diverse enough to evaluate these primers? I'm assuming yes, but wanted to make sure before I ran the qPCRs. If yes, I can run these sometime this week.

qPCR results here:

https://robertslab.github.io/sams-notebook/2020/08/24/qPCR-P.generosa-RPL5-v2-v3-and-TIF3s6b-v2-v3-Primer-Tests.html

**Comment by shellytrigg** [comment]
That's great! The numbered samples are diverse enough. The STAR/CHEW samples were sex unknown and just thrown in to see if we could identify them. Also, they were more recent hemolymph samples than the numbered samples but no need to remake cDNA for those.

**Comment by kubu4** [comment]
Unfortunately, those primers won't work as normalizing genes. RPL5 set has very wide spreads across samples, TIF3s6b doesn't amplify in most samples.

Notebook: https://robertslab.github.io/sams-notebook/2020/08/25/qPCR-P.generosa-RPL5-and-TIF3s6b-v2-and-v3-Normalizing-Gene-Assessment.html

**Comment by shellytrigg** [comment]
@kubu4 I just added 8 new primer pairs to the primer DB for amplifying the normalization genes 28S rRNA gene and Elongation Factor 1 alpha. These were designed by @SamGurr here: https://github.com/SamGurr/Pgenerosa_primers/tree/master/qPCR. Once these are arrive, can we test these on the pilot samples? If we don't have anymore cDNA available, maybe we can make some more from the RNA preps? Alternatively (and maybe better) we can do RNA preps and make cDNA from our real samples from the 2019-2020 pH exposure experiment.

**Comment by kubu4** [comment]
I ordered the primers.

I'm happy to make some more cDNA from Kaitlyn's samples (which is probably needed) to test the new primers. Any idea where the RNA might be (obvs, -80<sup>o</sup>C, but specific location)?

If you want to tackle the "real" samples, I'd be happy to make cDNA/run qPCRs after you isolate/quantify the RNA.

**Comment by shellytrigg** [comment]
Kaitlyn's RNA is in the -80C Rack 3, column 2, row, 2 and labeled "RNA isolation; geoduck 12/17" [-80C map](https://docs.google.com/spreadsheets/d/1Qsvz3QTURlPF_hX05BQxjom3484WuMfqQ1ILl9LEljU/edit?usp=sharing). Here are her notebook posts about the extractions:

- https://genefish.wordpress.com/2020/02/05/kaitlyns-notebook-geoduck-hemolymph-next-steps-and-rna-extractions/

- https://genefish.wordpress.com/2019/12/17/kaitlyns-notebook-geoduck-rna-extraction/

She did two rounds, one in December and one in February to make more for samples that were low from December. The february round (yellow tubes) she noted she did not dnase treat.

**Comment by kubu4** [comment]
Forgot to post, but I [tested out those 28s/EF1a primer sets](https://robertslab.github.io/sams-notebook/2020/09/14/qPCR-Geoduck-Normalizing-Gene-Primer-Checks.html). I then [picked one set from each group to test out on individual samples](https://robertslab.github.io/sams-notebook/2020/09/16/qPCR-Geoduck-Normalizing-Gene-Primers-28s-v4-and-EF1a-v1-Tests.html). I tested them on a set of Kaitlyn's original cDNA; I did _not_ make fresh cDNA yet.

If we ignore the "shoulder" in the 28s-v4 primer set, we still encounter the issue that the samples don't show consistent Cq values across all of the samples with either primer set.

I'll go ahead and do the following:

- re-quantify Kaitlyn's RNA

- make new cDNA

- re-run the 28s/EF1a primers on the new cDNA.

**Comment by shellytrigg** [comment]
sounds good @kubu4 . I still need to get into the lab a prep some of the new samples in case this behavior we keep observing is specific to these old pilot samples

**Comment by kubu4** [comment]
@shellytrigg I went in this morning to make some fresh cDNA from Kaitlyn's original RNA and realized that some samples had been DNased, whilte others have _maybe_ been DNased? It's not entirely clear from her notebook entries whether or not all the samples have been DNased. She indicates that samples will _need_ to be DNased before reverse transcription, but I can't find a notebook entry indicating that happened. Overall, it's kind of a mess and I don't think it's worthy additional time/effort to try to decipher.

So, I've decided it'll just be easier to keep track of things if you get new RNA isolated from your real samples, DNased, and quantified. Then, I'll take the samples from there, confirm they're free of DNA and get primers tested.
